openmdao/utils/general_utils.py

```diff
@@ -994,6 +997,29 @@ def _is_slice(indices):
     return any(isinstance(i, slice) for i in indices)
 
 
+def _is_ellipsis(indices):
+    """
+    Check if an array of indices contains an ellipsis special constant.
+
+    Parameters
+    ----------
+    indices : ndarray
+        Ellipsis indices to check.
+
+    Returns
+    -------
+    bool
+        Returns True if indices contains an ellipsis.
+    """
+    if indices is not None:
+        if hasattr(indices, "dtype") and indices.dtype == object:
+            return any(i == ... for i in indices)
+        elif isinstance(indices, tuple):
+            return any(i == ... for i in np.array(indices))
+    else:
+        return False
+
+
 def _slice_indices(slicer, out_size, out_shape):
     """
     Check if an array of indices contains a slice object.
```

Comment:
I think you should also test this in MPI (see `TestGroupMPISlice`) to make sure the data transfer works there.